backends/platform/sdl/morphos/morphos-main.cpp

```diff
@@ -30,24 +30,24 @@
 
 int main(int argc, char *argv[]) {
 
-	// Set up a stack cookie to avoid crashes from a stack set too low
-	static const char *stack_cookie __attribute__((used)) = "$STACK: 2048000";
+	// Set a stack cookie to avoid crashes from a too low stack.
+	static const char *stack_cookie __attribute__((used)) = "$STACK: 4096000";
```

Comment:
Why do you need to double the size of the stack cookie?

Reply:
to do same as AmigaOS... (https://github.com/scummvm/scummvm/pull/5765)
not really sur if necessary